packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -139,6 +141,18 @@ function is_static_element(node) {
 		}
 
 		if (attribute.value !== true && !is_text_attribute(attribute)) {
+			// in this situation we are already inlining the binding in the template
+			// so even if it's not a text attribute we can consider the element static
+			if (
```

Comment:
should this share the logic from here?

https://github.com/sveltejs/svelte/blob/1e5a385c968afb302beb38e7d8dd2982fdd66d0c/packages/svelte/src/compiler/phases/3-transform/client/visitors/RegularElement.js#L584

Reply:
Uh yeah absolutely...will fix

Reply:
Fixed it